bb-email/src/main/java/com/cognifide/qa/bb/email/EmailDataFactory.java

```diff
@@ -45,11 +44,11 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(EmailDataFactory.class);
 
-  private final Pattern addressPattern;
```

Comment:
Why is `final` keyword removed here?
For me it looks like a hint for developer that this is not mutable (and needs to be set in constructor).
Could we have it restored to `final` ?
(should have).